src/NuGet.Core/NuGet.Build.Tasks/NuGet.targets

```diff
@@ -496,7 +497,7 @@ Copyright (c) .NET Foundation. All rights reserved.
     ============================================================
   -->
   <Target Name="_GetRestoreSettingsCurrentProject"
-    Condition=" '$(TargetFrameworks)' == '' AND '$(RestoreProjectStyle)' == 'PackageReference' "
+    Condition=" '$(TargetFrameworks)' == '' AND ( '$(RestoreProjectStyle)' == 'PackageReference' OR '$(RestoreProjectStyle)' == 'DotnetToolReference' ) "
```

Comment:
PackageReferenceCompatibleProjectStyle

Reply:
The scope of that condition is local.

Reply:
as long as the target in which it is defined is executed before this one, that property can be used. it's much easier to maintain if there is only one place to add a future PackageReference compatible project style

Reply:
Refactored it. 

I was more comfortable to just add it to the _GetRestoreProjectStyle returns list.